router/core/factoryresolver.go

```diff
@@ -84,11 +85,18 @@ func NewDefaultFactoryResolver(
 		factoryLogger = abstractlogger.NewZapLogger(log, abstractlogger.DebugLevel)
 	}
 
+	var netPollConfig graphql_datasource.NetPollConfiguration
+
+	netPollConfig.ApplyDefaults()
+
+	netPollConfig.Disable = !enableNetPoll
```

Comment:
Would it have made more sense for this field to be the same as the option? i.e., `netPollConfig.Enable`? Was the logic already written for Disable?

Also, IMO, I prefer boolean state representations to be an adjective, i.e., `DisableD` (or `EnableD`).

Reply:
This code existed already. I like you thoughts here but it would be more efficient to review only the code I touched. We can address your points in a refactor. Nonetheless, I addressed the points because it was easy to do.

Reply:
I just wanted to write all my thoughts here. I didn't request any changes, so making no changes would also be understandable.